_slides/08-mgrinstein.md

```diff
@@ -3,7 +3,8 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+## bonjour !
+[My GitHub profile](https://github.com/mgrinstein/)
 
-![minertocat](https://octodex.github.com/images/minertocat.png)
+![notocat](https://octodex.github.com/images/notocat.jpg)
```

Comment:
I was wondering if it is a preference ? :)

Reply:
not a blocker ofc

Reply:
@DlnyaMazhari yes, so it matches my profile pic :)